altos-rust/port/cortex-m0/src/peripheral/usart/control.rs

```diff
@@ -0,0 +1,426 @@
+// Daniel Seitz and RJ Russell
+
+use super::super::Register;
+use super::defs::*;
+
+/// Three USART control registers.
+#[derive(Copy, Clone)]
+pub struct UsartCR {
```

Comment:
I feel UsartCR is more descriptive. If anything I think changing this to UsartCRx is more descriptive than UsartControl.